solenspipe/solenspipe.py

```diff
@@ -426,32 +426,50 @@ def qfunc(self,alpha,X,Y):
                          xfTalm=X[0],xfEalm=X[1],xfBalm=X[2])[polcomb][0]
 
     def qfunc_bh(self,alpha,X,Y,ils,blens,bhps,Alpp,A_ps):
-        polcomb=alpha
+
+        
+        """
```

Comment:
Thanks for adding these @qujia7! I think a good convention for the documentation is to have two sections, "Parameters" (describing the arguments of the function) and "Returns" (describing what the function returns). Each of the descriptions (of a given parameter or return) should indicate the type (e.g. int, float, np.array, dict...) as well describing what thing is at a more physics level (e.g. "lensing response function" as you've done already). The former is important, because it may be ambiguous what to actually provide as e.g. `blens` e.g. a numpy array at some set of ls, or an interpolation function etc.